src/server/common/components/phaseBanner/template.test.js

```diff
@@ -0,0 +1,26 @@
+import { renderComponent } from '~/src/server/common/test-helpers/component-helpers.js'
+
+describe('Phase Banner Component', () => {
+  /** @type {CheerioAPI} */
+  let $phaseBanner
+
+  beforeEach(() => {
+    $phaseBanner = renderComponent('phaseBanner', {
+      phase: 'Private beta'
+    })
+  })
+
+  test('Should render app phase banner component', () => {
+    expect($phaseBanner('[data-testid="app-phase-banner"]')).toHaveLength(1)
+  })
+
+  test('Should contain expected heading', () => {
+    expect($phaseBanner('[data-testid="app-phase"]').text().trim()).toBe(
+      'Private beta'
+    )
+  })
+})
+
+/**
+ * @import { CheerioAPI } from 'cheerio'
+ */
```

Comment:
Not 100% sure what this comment is for?  Presumably it's somewhere in the template already, but would be nice to clean up.